policy-library/policies/templates/gcp_vpc_sc_ensure_services_v1.yaml

```diff
@@ -74,6 +74,6 @@ spec:
         	count(missing_services) > 0
 
         	message := sprintf("Required services %v missing from service perimeter: %v.", [concat(", ", missing_services), asset.service_perimeter.name])
-        	metadata := {"resource": asset.name, "service_perimeter_name": asset.service_perimeter.name, "missing_services": missing_services}
+        	metadata := {"resource": asset.name, "restricted_service_perimeter_name": asset.service_perimeter.name, "missing_services": missing_services}
```

Comment:
do not change any of the policies they are general, not related to the foundation code